blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,474 @@
+# -*- coding: utf-8 -*-
+"""This module contains a MindMeld application"""
+from mindmeld import Application
+from mindmeld.core import FormEntity
+import random
+import json
+
+app = Application(__name__)
+
+__all__ = ["app"]
+
+# User data and helper functions for handlers
+
+user_data = {}
+
+sample_users = {
+    "johndoe123": 0,
+    "larry_l12": 1,
+    "splashbro30": 2,
+}
+
+
+def _pull_data(request):
+    with open("banking_blueprint/data/sample_user_data.json") as f:
+        global user_data
+        data = json.load(f)
+        if request.context.get("user_name"):
+            user_name = request.context.get("user_name")
+            user = sample_users[user_name]
+        else:
+            user = random.choice([0, 1, 2])
+
+        user_data = data[user]
+
+
+def _get(key):
+    return user_data[key]
+
+
+def _put(key, value):
+    global user_data
+    user_data[key] = value
+
+
+def _check_value(responder):
+    return _get(responder.slots["origin"]) >= responder.slots["amount"]
+
+
+def _credit_amount_helper(request, responder, entity):
+    responder.slots["payment"] = (
+        entity["value"][0]["cname"] if len(entity["value"]) > 0 else entity["text"]
+    )
+    if responder.slots["payment"] == "minimum":
+        responder.reply(
+            "Ok we have scheduled your credit card payment for your {payment} balance of $"
```

Comment:
```suggestion
            "OK, we have scheduled your credit card payment for your minimum balance of $"
```
(since you're already adding "balance", might as well not compute it)